Source/Initialization/PlasmaInjector.H

```diff
@@ -21,6 +21,10 @@
 #include <AMReX_ParmParse.H>
 #include <AMReX_Utility.H>
 
+#ifdef WARPX_USE_OPENPMD
```

Comment:
Maybe only include this in `PhysicalParticleContainer.cpp` so far, since you don't seam to use openPMD classes in this header yet.

Reply:
Thanks @ax3l ! That makes sense, I will fix it now.